src/components/Menu.js

```diff
@@ -0,0 +1,20 @@
+import { NavLink, useLocation } from 'react-router-dom';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
import { NavLink, useLocation } from "react-router-dom";
```